plugins/modules/proxmox_cluster_ha_groups.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2025, Markus Kötter <koetter@cispa.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_cluster_ha_groups
+
+short_description: Management of HA groups in roxmox VE Cluster
+
+version_added: "1.0.2"
+
+description: 
+  - Configure HA groups via /cluster/ha/groups
```

Comment:
Generally all descriptions (except `short_description`) should have paragraphs with full sentences.
```suggestion
  - Configure HA groups via C(/cluster/ha/groups).
```
Also note that you should use Ansible markup (see https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation and the sections below it for details).